community/samples/serving/helloworld-r/HelloWorld.R

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/Rscript
+#!/bin/Rscript
```

Comment:
Is the capitalization normal? This is the kind of thing that tends to have OS-level inconsistencies.

Reply:
Yes, this is standard R.